ops/model.py

```diff
@@ -1094,13 +1094,43 @@ def _validate_content(cls, content: Optional[Dict[str, str]]):
 
     @property
     def id(self) -> Optional[str]:
-        """Unique identifier for this secret.
+        """Locator ID (URI) for this secret.
+
+        This has an unfortunate name for historical reasons, as it's not
+        really a unique identifier, but the secret's locator URI, which may or
+        may not include the model UUID (for cross-model secrets).
```

Comment:
We could consider moving all the logic from this method into a new method called `uri`, and just leave this as a stub that calls `self.uri` and logs a deprecation warning for now?